app/src/main/java/org/wildaid/ofish/ui/tabs/TabsFragmentHost.kt

```diff
@@ -144,13 +144,18 @@ class TabsFragmentHost : Fragment(R.layout.fragment_tabs), OnNextClickedListener
                 }
             } else {
                 val visitedStatusColorRes =
-                    if (tabState.status == TabStatus.SKIPPED) R.color.tabs_amber else R.color.main_blue
+                    if (tabState.status == TabStatus.SKIPPED) R.color.orange else R.color.tab_text_color
+
+                val bgVisitedStatusColorRes =
+                    if (tabState.status == TabStatus.SKIPPED) R.color.orange else R.color.blue
+
                 val visitedStatusColor = resources.getColor(visitedStatusColorRes, null)
+                val bgVisitedStatusColor = resources.getColor(bgVisitedStatusColorRes, null)
 
                 tabs_layout.getTabAt(tabState.position)?.customView?.also { view ->
                     view.findViewById<View>(R.id.tab_bottom_line)?.setVisible(true)
                     view.findViewById<View>(R.id.tab_bottom_line)
-                        ?.setBackgroundColor(visitedStatusColor)
+                        ?.setBackgroundColor(bgVisitedStatusColor)
```

Comment:
I'm confused as to the difference between when visitedStatusColor is used and bgVisitedStatusColor....are we using visitedStatusColor anywhere else?

Reply:
In the previous version blue color is used for Tab text in both light and dark mode and the same for bg(indicator color) but the screenshots has blue color for light mode and white color for Tab text and different color code of blue bg(indicator) for bg(indicator) thats why seprated both text and indicator colors.

Reply:
Yes, I can see the next line after that is where visitedStatusColor is used:

```
                    view.findViewById<View>(R.id.tab_bottom_line)
                        ?.setBackgroundColor(bgVisitedStatusColor)
                    view.findViewById<CheckedTextView>(android.R.id.text1)
                        ?.setTextColor(visitedStatusColor)
```

So we ARE still using visitedStatusColor, for the text itself.